server/misc.go

```diff
@@ -33,7 +33,7 @@ func (s *Server) Translate(ctx context.Context,
 	}
 
 	out := pb.TranslateResponse{}
-	mappings, err := translator.ParseMapping(in.GetSchemaMapping(), s.metadata.Bq)
+	mappings, err := translator.ParseMapping(in.GetSchemaMapping(), "bq")
```

Comment:
For my understanding, where is the DB used in the mapping (I tried following the func calls, it went multiple levels :)

Reply:
It's passed into here: https://github.com/datacommonsorg/mixer/blob/master/translator/mcf.go#L129

where it adds the bq to the column name prefix